internal/local/app.go

```diff
@@ -259,10 +264,6 @@ func LoadAppConfig(path string) (App, error) {
 	if !appOK {
 		return App{}, nil
 	}
-
-	if err := app.LoadConfig(); err != nil {
-		return App{}, err
-	}
 	return app, nil
```

Comment:
what do you think about removing the `LoadAppConfig` function entirely?  It's not doing much besides handling the `!ok` case of `FindApp`.  considering there's only a handful of usages at this point, it seems like a good chance to reduce the exported surface area of our `local` package

Reply:
these methods are fairly intertwined: `Load()`, `LoadConfig()`, `LoadApp()`, `LoadAppConfig()`
I think we could survive with just: `LoadApp()` and `LoadConfig()` [called by `LoadApp()` and `FindApp()`]

pushed a commit with the changes, lmk what you think!